Task5/css/header.css

```diff
@@ -0,0 +1,114 @@
+body {
+    width: 1200px;
+    margin: 0 auto;
+}
+
+header {
+    height: 250px;
+    width: 1200px;
+    overflow: hidden;
+    position: relative;
+    z-index: 0;
+    display: flex;
+    align-items: center;
+    justify-content: center;
+}
+
+.signboard {
+    width: 370px;
+    height: 100px;
+    display: flex;
+    align-items: center;
+    justify-content: center;
+    position: absolute;
+    background: #ffffff;
+    z-index: 1;
+    font-size: 35px;
+    font-style: italic;
+    padding: 4px;
+}
+
+.signboard::before {
+    content: "";
+    position: absolute;
+    z-index: -1;
+    background: #ffffff;
+    border: 3px solid black;
+    width: 360px;
+    height: 90px;
+}
+
+.inner {
+    width: 350px;
+    height: 80px;
+    border: 2.5px solid;
+    padding: 0;
+}
+
+.social {
+    transform: rotate(-1.7grad);
+    height: 300px;
+    width: 1350px;
+}
+
+.icon {
+    display: inline-block;
```

Comment:
`display: inline-block;` - a redundant property. Isn't it?

Reply:
you right! fixed.